src/main/java/Duke.java

```diff
@@ -1,10 +1,160 @@
+import java.util.Scanner;
+
+/**
+ * Represents the bot Duke.
+ * Duke can perform tasks based on user input.
+ *
+ * {@value #NUM_OF_TASKS} Number of Tasks.
+ */
 public class Duke {
+
+    private static int NUM_OF_TASKS = 0;
+    private static Task[] list = new Task[100];
```

Comment:
Consider using a constant variable to represent "100" so it isn't a magic number.